pkg/api/k8s_versions.go

```diff
@@ -30,7 +30,7 @@ var k8sComponentVersions = map[string]map[string]string{
 		"aci-connector":                    "virtual-kubelet:latest",
 		ContainerMonitoringAddonName:       "oms:ciprod01092019",
 		AzureCNINetworkMonitoringAddonName: "networkmonitor:v0.0.6",
-		"cluster-autoscaler":               "cluster-autoscaler:v1.14.0",
+		"cluster-autoscaler":               "cluster-autoscaler:v1.14.2",
```

Comment:
We were already using this cluster-autoscaler for Kubernetes 1.14.2, this just bumps it for 1.15.0-alphas (which don't have a specific CA release yet).